frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -367,6 +374,26 @@ local footerTextGeneratorMap = {
             end
         end
     end,
+    page_turning_inverted = function(footer)
+        local symbol_type = footer.settings.item_prefix
+        local prefix = symbol_prefix[symbol_type].page_turning_inverted
+        if G_reader_settings:isTrue("input_invert_page_turn_keys") or G_reader_settings:isTrue("input_invert_left_page_turn_keys") or
+           G_reader_settings:isTrue("input_invert_right_page_turn_keys") or G_reader_settings:isTrue("inverse_reading_order") then
+            if symbol_type == "icons" or symbol_type == "compact_items" then
+                return symbol_prefix.icons.page_turning_inverted
+            else
+                return T(_("%1".. "I"), prefix)
```

Comment:
I'm not sure this is right, @Frenzie ?
We probably don't need the `..`, you could just use `"%1I"`.
Do we need this to be translatable `T()` ? (I see above it's use for on/off).

Reply:
I for inverted seems like a particular language (or am I misreading this) so I'd lean toward translation? But yes, no .. needed, quite the opposite.

Reply:
Yes, I for inverted. And now N for ‘normal’ i guess

Reply:
OK with `N`  meaning Normal or No (but also Negated).
Isn't `R` more international sounding as Reverted/Reversed than `I` (for Inverted, but also the positive sounding Initial, In) ?

Reply:
The complication is called “ Page turning inverted”, i am sure people will figure it out. You just might be overthinking it now.